### PR TITLE
Setup reviewdog github action

### PR DESCRIPTION
**Pull request by neriyarden**
WIP

### CODE REVIEW

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: reviewdog
+on: pull_request
+
+jobs:
+  eslint:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (unicop) [inline .github/workflows/reviewdog.yml:9]: please add cache support
https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#example-using-the-cache-action

Comment (unicop) [reply .github/workflows/reviewdog.yml:9]: right after the checkout you can add
```suggestion
      - uses: actions/checkout@v2
      
      - name: Cache node modules
        id: cache-npm
        uses: actions/cache@v3
        env:
          cache-name: cache-node-modules
        with:
          # npm cache files are stored in `~/.npm` on Linux/macOS
          path: ~/.npm
          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
          restore-keys: |
            ${{ runner.os }}-build-${{ env.cache-name }}-
            ${{ runner.os }}-build-
            ${{ runner.os }}-
```

**Comment by neriyarden** [comment]
[[Github] Setup reviewdog github action for commenting lint errors on PRs](https://trello.com/c/kZHE5eS1/26-github-setup-reviewdog-github-action-for-commenting-lint-errors-on-prs)

**Comment by neriyarden** [comment]
duplicate

**Pull request closed without merging**